smpe/bld/service/current-hold.txt

```diff
@@ -0,0 +1,20 @@
+****************************************************************
+* Affected function: Zowe z/OSMF pre req                       *
+****************************************************************
+* Description: Install required z/OSMF PTF                     *
+****************************************************************
+* Timing: post-APPLY                                           *
```

Comment:
I would mark this as pre-APPLY. If the zOSMF PTFs are applied after the Zowe PTF, Zowe is broken until the PTFs are applied, and Zowe likely needs another server restart so we would notice the zOSMF update.